e2e/pages/home.page.js

```diff
@@ -0,0 +1,10 @@
+var HomePage = function () {
+    
+    this.greetingHeader = $('main.container h1');
```

Comment:
Should I open a bug related to assigning ids/attributes for all valuable elements or mark this comment as fixed?